### PR TITLE
Added import of quantized modules so they get registered

### PR DESCRIPTION
**Pull request by spsancti**

### CODE REVIEW

**Review by shaydeci — APPROVED** [review]
LGTM

**Review by shaydeci — CHANGES_REQUESTED** [review]
**Sorry for the approve before.

We need to talk this over, since PyTorch quantization is an optional installation.

File: src/super_gradients/modules/__init__.py [hunk 1]
```diff
@@ -17,3 +17,30 @@
...
+        "QuantBottleneck",
+        "QuantResidual",
+        "QuantSkipConnection",
+        "QuantCrossModelSkipConnection",
+        "QuantBackboneInternalSkipConnection",
+        "QuantHeadInternalSkipConnection",
+    ]
+
+except (ImportError, NameError, ModuleNotFoundError) as import_err:
+    logger.warning(f"Failed to import pytorch_quantization: {import_err}")
```
Comment (ofrimasad) [inline src/super_gradients/modules/__init__.py:41]: please change this to debug (not warning)

**Review by ofrimasad — COMMENTED** [review]
minor comment

**Review by ofrimasad — APPROVED** [review]
LGTM

**Review by shaydeci — APPROVED** [review]

**Pull request merged**